src/tests/ftest/util/agent_utils.py

```diff
@@ -88,13 +88,14 @@ def run_agent(basepath, server_list, client_list=None):
 
     for client in client_list:
         sessions[client] = subprocess.Popen(
-            ["ssh", client, "{} -i".format(daos_agent_bin)],
+            ["ssh", client, "-o ConnectTimeout=10",
```

Comment:
Ah, reading the manpage, this includes the SSH handshake now, not just the TCP connection.  Good thing we decided on 10s.